src/calibrationlib/operator_feature_export.cpp

```diff
@@ -0,0 +1,107 @@
+#include <regex>
+
+#include <expression/expression_utils.hpp>
+#include <expression/lqp_column_expression.hpp>
+#include <logical_query_plan/stored_table_node.hpp>
+#include <utils/assert.hpp>
+#include "csv_writer.hpp"
+#include "hyrise.hpp"
+#include "operator_feature_export.hpp"
+#include "storage/table.hpp"
+
+namespace opossum {
+
+OperatorFeatureExport::OperatorFeatureExport(const std::string& path_to_dir) : _path_to_dir(path_to_dir) {}
+
+void OperatorFeatureExport::export_to_csv(std::shared_ptr<const AbstractOperator> op) const {
+  if (op != nullptr) {
```

Comment:
You could also write `if (op)` which would have the same effect. However, should that function be ever called intentionally with a `nullptr`? If not, I would put an Assert here.